src/SignalR/clients/cpp/test/gtest/gtest.vcxproj

```
@@ -19,7 +19,7 @@
     </ProjectConfiguration>
   </ItemGroup>
   <ItemGroup>
-    <ClCompile Include="googletest\googletest\src\gtest-all.cc" />
+    <ClCompile Include="..\..\..\..\..\submodules\googletest\googletest\src\gtest-all.cc" />
```

```suggestion
    <ClCompile Include="$(GoogleTestSubmoduleRoot)googletest\src\gtest-all.cc" />
```

Suggestion: use this variable: https://github.com/aspnet/AspNetCore/blob/5a75084826f21743e4395966bd52e5d1a00e9f70/Directory.Build.props#L47

---

Yeah, couldn't figure out how to make it work, probably something to do with how separated the C++ build is currently.

---

It's probably because src/SignalR/ has a broken chain of Directory.Build.props files. https://github.com/aspnet/AspNetCore/blob/master/src/SignalR/Directory.Build.props doesn't import the root level Directory.Build.props file. That's one of the problems fixed by https://github.com/aspnet/AspNetCore/pull/6457